python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -94,16 +87,14 @@ def makeDataRefList(self, namespace):
                     if overlapsTract(tract, wcs, box):
                         visitTract[visit].add(tract.getId())
             else:
-                tract = dataId.pop("tract")
                 # making a DataRef for src fills out any missing keys and allows us to iterate
-                for ref in namespace.butler.subset("src", dataId=dataId):
-                    self._addDataRef(namespace, ref.dataId, tract)
+                self.refList.extend(ref for ref in namespace.butler.subset(self.datasetType, dataId=dataId))
```

Comment:
You're not doing this on `"src"` now.  If that's deliberate, remove the comment.